WORKSPACE

```diff
@@ -8,7 +8,9 @@ load("//bazel:api_repositories.bzl", "envoy_api_dependencies")
 
 envoy_api_dependencies()
 
-load("//bazel:repositories.bzl", "envoy_dependencies")
+load("//bazel:repositories.bzl", "envoy_dependencies", "envoy_repo")
+
+envoy_repo()
```

Comment:
call this in `repositories.bzl`, like https://github.com/envoyproxy/envoy/blob/main/bazel/repositories.bzl#L104

Reply:
i didnt do that as its not a dependency, but i can move

Reply:
moved